lambdas/service/app.py

```diff
@@ -957,7 +959,59 @@ def generate_manifest(event, context):
     return {'Location': presigned_url}
 
 
-@app.route('/dss/files/{uuid}', methods=['GET'], cors=True)
+dss_files_spec = {
+    'tags': ['DSS'],
+    'parameters': [
+        params.path('uuid', str, description='UUID of the file to be checked out'),
+        params.query('fileName',
+                     schema.optional(str),
+                     description='The desired name of the file. If absent, the UUID of the file will be used.'),
+        params.query('requestIndex',
+                     schema.optional(int),
+                     description='Number of attempts made through the endpoint to fetch the desired file.'),
+        params.query('wait',
+                     schema.optional(int),
+                     description='If this parameter is 1 and the checkout is still in process, the server will wait '
```

Comment:
Wouldn't a triple quoted string be easier here? If indention and newlines are a problem, add a helper to post-process the string.